src/components/Beneficiaries/BeneficiariesAddModal.tsx

```diff
@@ -343,4 +344,4 @@ const mapDispatchToProps: MapDispatchToProps<DispatchProps, {}> = dispatch => ({
 });
 
 // tslint:disable-next-line:no-any
-export const BeneficiariesAddModal = injectIntl(connect(mapStateToProps, mapDispatchToProps)(BeneficiariesAddModalComponent) as any);
+export const BeneficiariesAddModal = injectIntl(connect(mapStateToProps, mapDispatchToProps)(BeneficiariesAddModalComponent) as any) as any;
```

Comment:
same as above

Reply:
for some cases it works and for some not
anyway we are gonna migrate it